pkg/crc/machine/start.go

```diff
@@ -811,7 +811,7 @@ func updateCockpitConsoleBearerToken(sshRunner *crcssh.Runner) error {
 		return fmt.Errorf("failed to write cockpit bearer token: %w", err)
 	}
 
-	if err := sshRunner.CopyData([]byte(token), "/home/core/cockpit-bearer-token", 0600); err != nil {
+	if err := sshRunner.CopyDataPrivileged([]byte(token), "/home/core/cockpit-bearer-token", 0600); err != nil {
```

Comment:
This can be changed to use `CopyData`, in the next lines we are again changing the ownership of this file to `core`

https://github.com/cfergeau/crc/blob/728f60ae630ef6a72eaa98a9d61c00e8f5d314af/pkg/crc/machine/start.go#L815-L822

Reply:
Oh nice, happy that there's a second user for CopyData/CopyDataPrivileged!